jupyter_releaser/util.py

```diff
@@ -247,8 +247,11 @@ def bump_version(version_spec, *, changelog_path="", version_cmd=""):
             version_cmd = version_cmd or TBUMP_CMD
 
         if PYPROJECT.exists():
-            if "tbump" in PYPROJECT.read_text(encoding="utf-8"):
+            pyproject_text = PYPROJECT.read_text(encoding="utf-8")
+            if "tbump" in pyproject_text:
                 version_cmd = version_cmd or TBUMP_CMD
+            elif "hatchling" in pyproject_text:
+                version_cmd = version_cmd or "hatch version"
```

Comment:
I have no context but you can also do `hatchling version` if you don't have the full CLI

Reply:
Oh good catch, we actually depend on `hatchling` and not `hatch` for this library.